duckdb/README.md

```diff
@@ -1,6 +1,26 @@
 # DuckDB Data Connector
 
-Follow these steps to get started with DuckDB (v0.10.1) as a Data Connector.
+Follow these steps to get started with DuckDB (v0.10.2) as a Data Connector.
+
+**Preparation** Install [DuckDB CLI](https://duckdb.org/docs/installation/?version=stable&environment=cli&platform=macos&download_method=package_manager). Create a persistent file-based DuckDB database by Providing a path to database file when starting DuckDB. DuckDB will create the database if there's no database file existing at the specified file path.
+
+```bash
+duckdb /data/myawesomedb.db
+```
+
+Run the following queries in DuckDB to generate TPC-H data with DuckDB's [TCPH extension](https://duckdb.org/docs/extensions/tpch.html)
+
+```SQL
+INSTALL tpch;
+LOAD tpch;
+CALL dbgen(sf = 1);
+```
+
+Run the following query to check the tables generated.
+
+```SQL
+SHOW TABLES;
+```
 
 **Step 1.** Edit the `spicepod.yaml` file in this directory. Replace the `open` params in the `duckdb_quickstart` dataset with the path to your DuckDB persistent File. Edit `[local_table_name]` to your desired name for the table and `[database].[schema].[table]` to the name of the table in your DuckDB instance.
```

Comment:
Can we please update `spicepod.yaml` so it matches tpch.db file location and schema so it just works w/p modifications.

Ideal quickstart is a list of simple steps you can just copy-paste to get it working.